board/gameBoard.go

```diff
@@ -0,0 +1,87 @@
+package board
+
+import (
+	"fmt"
+	"math/rand"
+	"reflect"
+)
+
+
+const color = "\033["
+const colorReset = "0m"
+var keyMap = map[string]int{
+	"G": 32,
+	"R": 31,
+	"Y": 33,
+	"B": 34,
+}
+
+var Keys = func () []string {
+	keys := reflect.ValueOf(keyMap).MapKeys()
+	var formatKeys []string
+	for _, key := range keys {
+		formatKeys = append(formatKeys, key.String())
+	}
+	return formatKeys
+}()
+
+
+type GameBoard struct {
+	board [][]string
+}
+
+func CreateBoard(size int) GameBoard {
+	//declare new board
+	newBoard := make([][]string, size, size)
+	ln := len(Keys)  //make sure that you only call len once
+	// initial values
+	for i := 0; i < size; i++ {
+		newRow := make([]string, size, size)
+		for y := 0; y < size; y++ {
+			newRow[y] = Keys[rand.Intn(ln)]
+		}
+		newBoard[i] = newRow
+	}
+	return GameBoard{board: newBoard}
+}
+
+func (g GameBoard) PrintBoard() {
```

Comment:
See the String() method option for go. 
It allows you to determinate how a `fmt.Println(instance)` will look like. 

Replace the PrintBoard method with a String() method representation